ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -115,6 +121,7 @@ class LeftNavMenuViewController: UIViewController {
             let statusCellChild = MSFListCellState()
             statusCellChild.leadingViewSize = .small
             statusCellChild.title = presence.cellTitle()
+            statusCellChild.cellBackgroundColor = UIColor.clear
```

Comment:
The cell should have the elevation color the same way as the drawer, shouldn't it? (can you follow up with @sophialee0416 on this?)
There is a weird animation effect going on probably caused by this change.
The button action does not seem to be triggered either.

https://user-images.githubusercontent.com/68076145/111937584-46eaab80-8a85-11eb-815e-830ad1d0f610.mov

Reply:
The clear color is set for dark mode. I have resolved the problem with button action but I am not specifically sure about the animation effect. Sophia also confirmed there is no elevation color and I couldn't find anything on Figma